.gitignore

```diff
@@ -11,6 +11,7 @@ bin/
 .env
 .coverprofile
 coverage.txt
+vhd-publishing-info.json
```

Comment:
will this cover any and all sub-dirs as well?

Reply:
addressed